ParallaxGenLib/include/ParallaxGenConfig.hpp

```diff
@@ -17,9 +17,10 @@ class ParallaxGenConfig {
   std::filesystem::path ExePath;
 
   // Config Structures
-  std::unordered_set<std::wstring> NIFBlocklist;
-  std::unordered_set<std::wstring> DynCubemapBlocklist;
-  std::unordered_map<std::filesystem::path, NIFUtil::TextureType> ManualTextureMaps;
+  std::unordered_set<std::wstring> NIFBlocklist {};
+  std::unordered_set<std::wstring> DynCubemapBlocklist{};
+  std::unordered_map<std::filesystem::path, NIFUtil::TextureType> ManualTextureMaps{};
+  std::set<std::wstring> BSAExclusionList{};
```

Comment:
Does this need to be a set? If we don't care about order we can make this an unordered_set so it becomes constant time lookups instead of log(n)

Reply:
For small number of elements, set is usually preferred and a bit simpler to read. Your decision

Reply:
Everything else is unordered set here so let's go with unordered_set. I agree the performance different will be negligible, but we can circle back when we clean stuff up later, let's set it the same as others for now.